txscript/standard_test.go

```diff
@@ -1013,3 +1013,28 @@ func TestStringifyClass(t *testing.T) {
 		}
 	}
 }
+
+// TestNullDataScript tests whether NullDataScript returns a valid script.
+func TestNullDataScript(t *testing.T) {
+	data := []byte{0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16,
```

Comment:
Please make this table driven so different data can be tested.

Something like the following

```Go
tests := []struct{
	name string
	data []byte
	valid bool
	expected []byte
}{
	{
		name: "more than max allowed",
		data: hexToBytes("000102....80"),
		valid: false,
		expected: nil,
	},
	{
		name: "small data",
		data: hexToBytes("01"),
		valid: false,
		expected: []byte{OP_RETURN, 0x51},
	},
	...
}

for i, test := range tests {
	... // Call func, check for err against valid, bytes.Compare(result, test.expected), etc
}
````